content/en/docs/Configuration/multi-cluster/external/_index.md

```diff
@@ -3,20 +3,27 @@ title: "External Kiali"
 description: "Deploy Kiali on a Management Cluster."
 ---
 
-Larger mesh deployments may desire to separate mesh operation from mesh observability. By deploying Kiali on a cluster away from the mesh, it allows for dedicated management and resource consumption.
+Larger mesh deployments may desire to separate mesh operation from mesh observability. This means deploying Kiali, and potentially other observability tooling, away from the mesh.
+
+This separation allows for:
+
+- Dedicated management of mesh observability
+- Reduced resource consumption on mesh clusters
+- Centralized visibility across multiple mesh clusters
+- Improved security isolation
 
 ### Deployment Model
 
 This deployment model requires a minimum of two clusters. The Kiali "home" cluster (where Kiali is deployed) will serve as the "management" cluster. The "mesh" cluster(s) will be where your service mesh is deployed. The mesh deployment will still conform to any of the Istio deployment models that Kiali already supports. The fundamental difference is that Kiali will not be co-located with an Istio control plane, but instead will reside away from the mesh. For multi-cluster mesh deployments, all of the same requirements apply, such as unified metrics and traces, etc.
 
-It is recommended, but not required, that joining Kiali on the "management" cluster, you also place your other observability tooling, like the metrics store. This will further reduce observability resources on the mesh cluster(s), and will likely reduce latency between Kiali and those data stores.
+It is recommended, but not required, that joining Kiali on the "management" cluster is other observability tooling, like the metrics store. This will further reduce observability resources on the mesh cluster(s), and will likely reduce latency between Kiali and those data stores. But, it may require additional work, like federating Prometheus databases, etc.
```

Comment:
`It is recommended, but not required, that joining Kiali on the "management" cluster **is** other observability tooling,...`

The word "is" here seems wrong. Do you mean "with" ???